pdr_backend/lake/duckdb_data_store.py

```diff
@@ -349,3 +352,105 @@ def get_query_move_table_data(self, from_table, to_table):
         return (
             f"INSERT INTO {to_table.table_name} SELECT * FROM {from_table.table_name};"
         )
+
+    @enforce_types
+    def export_tables_to_parquet_files(
+        self,
+        seconds_between_exports: int,
```

Comment:
I think this should be UnixTimeS to make it clearer. And later, instead of multiplying it by 1000, call to_ms